config/spec/mainnet.go

```diff
@@ -88,14 +87,17 @@ const (
 	// mainnetEVMInflationPerBlockDeneb1 is the amount of native EVM balance (in Gwei) to be
 	// minted to the EVMInflationAddressDeneb1 via a withdrawal every block in the Deneb1 fork.
 	mainnetEVMInflationPerBlockDeneb1 = 5.75 * params.GWei
+
+	// mainnetMinValidatorWithdrawabilityDelay is the number of epochs of delay epochs of delay for a balance to be withdrawable.
+	// 256 Epochs equates to roughly ~27 hours of withdrawal delay. This gives us room to emergency fork if needed.
+	mainnetMinValidatorWithdrawabilityDelay = 256
```

Comment:
nit: use default when its the same
```suggestion
	mainnetMinValidatorWithdrawabilityDelay = defaultMinValidatorWithdrawabilityDelay
```